src/main/resources/assets/viafabricplus/lang/en_us.json

```diff
@@ -81,6 +81,7 @@
   "visual_settings.viafabricplus.change_font_renderer_behaviour": "Change Font Renderer behavior",
   "visual_settings.viafabricplus.enable_sword_blocking": "Enable sword animation",
   "visual_settings.viafabricplus.enable_block_hit_animation": "Enable block hit animation",
+  "visual_settings.viafabricplus.enable_legacy_tablist": "Enable legacy tablist behavior",
```

Comment:
Move translation at the end of the visual_settings block (in language files, translation keys are not sorted by versions like in the setting classes). That's something I want to change in the future but for now it would be better to keep this layout.